functions-methods.js

```diff
@@ -9,8 +9,13 @@
 // getEmailDomain("t.mellink@novi.nl") geeft novi.nl
 // getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
 
+function getEmailDomain(email){
+    let mail1 = email.substring(email.indexOf("@"))
+    return mail1.replace("@","")
```

Comment:
Je zou dit nog iets kunnen versimpelen
```suggestion
    return email.substring(email.indexOf("@") + 1);
```